patterns/blueprint-vpc-lattice/cluster2/aurora.tf

```diff
@@ -0,0 +1,23 @@
+provider "random" {}
+
+module "rds-aurora" {
+  source  = "aws-ia/rds-aurora/aws"
+  version = "0.0.7"
+  # insert the 5 required variables here
+
+  password             = random_password.password.result
+  username = "admin"
```

Comment:
I had issue with admin because it is restricted, I used admins instead

Reply:
applied